recipes/disperse/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "disperse" %}
+{% set version = "v0.9.24" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/thierry-sousbie/DisPerSE/archive/master.tar.gz
+  sha256: 3406f08470df454bc5428d18ec4828b8b683719cc8eddde52a29b91639f51771
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - openmp  # [osx]
+  host:
+    - gsl
+    - cgal
+    - libopenblas
+    - gmp
+    - openmp  # [osx]
+    - _openmp_mutex  # [linux]
+
+test:
+  commands:
+    - mse
+    - fieldconv
+    - netconv
+    - skelconv
```

Comment:
This is an exhaustive list of all the binaries generated by the compilation step.